clients/interactions.py

```diff
@@ -32,6 +32,14 @@ async def send_commands_list(ctx):
     await ctx.send(embed=message)
 
 
+def update_current_squad_player_list(member, before, after):
+    """ Return True if the track question should be sent,
```

Comment:
Are these comments correct? What is this suppose to do? It doesn't update anything does it?